tests/test_cli/test_launch.py

```diff
@@ -122,6 +116,7 @@ def teardown_class(cls):
             pass
 
 
+@pytest.mark.unstable
```

Comment:
Why is this now marked as unstable? It has behaved stable on the recent builds..